src/liboslnoise/simplexnoise.cpp

```diff
@@ -70,52 +70,81 @@ static OSL_DEVICE float zero[] = { 0.0f, 0.0f, 0.0f, 0.0f };
-    {  0.0f,  1.0f, -1.0f }, {  0.0f, -1.0f, -1.0f }
-};
+static OSL_DEVICE float grad3lut[16][3]
+    = { { 1.0f, 0.0f, 1.0f },   { 0.0f, 1.0f, 1.0f },  // 12 cube edges
+        { -1.0f, 0.0f, 1.0f },  { 0.0f, -1.0f, 1.0f },
+        { 1.0f, 0.0f, -1.0f },  { 0.0f, 1.0f, -1.0f },
+        { -1.0f, 0.0f, -1.0f }, { 0.0f, -1.0f, -1.0f },
+        { 1.0f, -1.0f, 0.0f },  { 1.0f, 1.0f, 0.0f },
+        { -1.0f, 1.0f, 0.0f },  { -1.0f, -1.0f, 0.0f },
+        { 1.0f, 0.0f, 1.0f },   { -1.0f, 0.0f, 1.0f },  // 4 repeats to make 16
+        { 0.0f, 1.0f, -1.0f },  { 0.0f, -1.0f, -1.0f } };
 
 // Gradient directions for 4D
-static OSL_DEVICE float grad4lut[32][4] = {
-  { 0.0f, 1.0f, 1.0f, 1.0f }, { 0.0f, 1.0f, 1.0f, -1.0f }, { 0.0f, 1.0f, -1.0f, 1.0f }, { 0.0f, 1.0f, -1.0f, -1.0f }, // 32 tesseract edges
-  { 0.0f, -1.0f, 1.0f, 1.0f }, { 0.0f, -1.0f, 1.0f, -1.0f }, { 0.0f, -1.0f, -1.0f, 1.0f }, { 0.0f, -1.0f, -1.0f, -1.0f },
-  { 1.0f, 0.0f, 1.0f, 1.0f }, { 1.0f, 0.0f, 1.0f, -1.0f }, { 1.0f, 0.0f, -1.0f, 1.0f }, { 1.0f, 0.0f, -1.0f, -1.0f },
-  { -1.0f, 0.0f, 1.0f, 1.0f }, { -1.0f, 0.0f, 1.0f, -1.0f }, { -1.0f, 0.0f, -1.0f, 1.0f }, { -1.0f, 0.0f, -1.0f, -1.0f },
-  { 1.0f, 1.0f, 0.0f, 1.0f }, { 1.0f, 1.0f, 0.0f, -1.0f }, { 1.0f, -1.0f, 0.0f, 1.0f }, { 1.0f, -1.0f, 0.0f, -1.0f },
-  { -1.0f, 1.0f, 0.0f, 1.0f }, { -1.0f, 1.0f, 0.0f, -1.0f }, { -1.0f, -1.0f, 0.0f, 1.0f }, { -1.0f, -1.0f, 0.0f, -1.0f },
-  { 1.0f, 1.0f, 1.0f, 0.0f }, { 1.0f, 1.0f, -1.0f, 0.0f }, { 1.0f, -1.0f, 1.0f, 0.0f }, { 1.0f, -1.0f, -1.0f, 0.0f },
-  { -1.0f, 1.0f, 1.0f, 0.0f }, { -1.0f, 1.0f, -1.0f, 0.0f }, { -1.0f, -1.0f, 1.0f, 0.0f }, { -1.0f, -1.0f, -1.0f, 0.0f }
-};
+static OSL_DEVICE float grad4lut[32][4]
+    = { { 0.0f, 1.0f, 1.0f, 1.0f },
+        { 0.0f, 1.0f, 1.0f, -1.0f },
+        { 0.0f, 1.0f, -1.0f, 1.0f },
+        { 0.0f, 1.0f, -1.0f, -1.0f },  // 32 tesseract edges
+        { 0.0f, -1.0f, 1.0f, 1.0f },
+        { 0.0f, -1.0f, 1.0f, -1.0f },
+        { 0.0f, -1.0f, -1.0f, 1.0f },
+        { 0.0f, -1.0f, -1.0f, -1.0f },
+        { 1.0f, 0.0f, 1.0f, 1.0f },
+        { 1.0f, 0.0f, 1.0f, -1.0f },
+        { 1.0f, 0.0f, -1.0f, 1.0f },
+        { 1.0f, 0.0f, -1.0f, -1.0f },
+        { -1.0f, 0.0f, 1.0f, 1.0f },
+        { -1.0f, 0.0f, 1.0f, -1.0f },
+        { -1.0f, 0.0f, -1.0f, 1.0f },
+        { -1.0f, 0.0f, -1.0f, -1.0f },
+        { 1.0f, 1.0f, 0.0f, 1.0f },
+        { 1.0f, 1.0f, 0.0f, -1.0f },
+        { 1.0f, -1.0f, 0.0f, 1.0f },
+        { 1.0f, -1.0f, 0.0f, -1.0f },
+        { -1.0f, 1.0f, 0.0f, 1.0f },
+        { -1.0f, 1.0f, 0.0f, -1.0f },
+        { -1.0f, -1.0f, 0.0f, 1.0f },
+        { -1.0f, -1.0f, 0.0f, -1.0f },
+        { 1.0f, 1.0f, 1.0f, 0.0f },
+        { 1.0f, 1.0f, -1.0f, 0.0f },
+        { 1.0f, -1.0f, 1.0f, 0.0f },
+        { 1.0f, -1.0f, -1.0f, 0.0f },
+        { -1.0f, 1.0f, 1.0f, 0.0f },
+        { -1.0f, 1.0f, -1.0f, 0.0f },
+        { -1.0f, -1.0f, 1.0f, 0.0f },
+        { -1.0f, -1.0f, -1.0f, 0.0f } };
 
```

Comment:
This was more readable prior to clang-format.

Reply:
fixing